test/test_unit_conjur_resource.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+from conjur.resource import Resource
+
+class ConjurResourceDataTest(unittest.TestCase):
+
+    def test_full_id_returns_pair_properly(self):
+        mock_resource = Resource(type_="sometype", name="somename")
+        resource_pair = mock_resource.full_id()
+
```

Comment:
Remove empty line

Reply:
did this purpose for readability. Otherwise tests are just a giant glob. But if you think this looks out of place i will remove